utils/_context/_scenarios/k8s_lib_injection.py

```diff
@@ -17,6 +25,170 @@
 from .core import Scenario
 
 
+class K8sScenario(Scenario):
+    """Scenario that tests kubernetes lib injection"""
+
+    def __init__(
+        self,
+        name,
+        doc,
+        github_workflow=None,
+        scenario_groups=None,
+        use_uds=False,
+        with_admission_controller=True,
+        weblog_env={},
+        dd_cluster_feature={},
+    ) -> None:
+        super().__init__(name, doc=doc, github_workflow=github_workflow, scenario_groups=scenario_groups)
+        self.use_uds = use_uds
+        self.with_admission_controller = with_admission_controller
+        self.weblog_env = weblog_env
+        self.dd_cluster_feature = dd_cluster_feature
+        self._tested_components = {}
+
+    def configure(self, config):
+        self.k8s_weblog = config.option.k8s_weblog
```

Comment:
Is there any reason to have the build step inside the run step ? in theory it should ba fast. But in practice, it's not that fast ...

Reply:
ok, if I understand correctly, you don't need to build it each time, because the tracer will be injected, right ?

Reply:
yes!